test_manager/main.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"strings"
+	"sync"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"guest-test-infra/test_manager/e2etest"
+)
+
+var (
+	workProject    = flag.String("work_project", "", "project to perform the work in, passed to Daisy as workflow project, will override WorkProject in template")
+	gcsPath        = flag.String("gcs_path", "", "GCS bucket to use, overrides what is set in workflow")
+
+	testName       = flag.String("test_name", "", "the E2E test name used for logging and daisy")
+	tarballPath    = flag.String("tarball_path", "", "a list of image tarball path that want to test, split by ',', at least one provided")
+	imageName      = flag.String("image_name", "", "a list of image that target image want to test, split by ',', at least one provided")
+	imageFamily    = flag.String("image_family", "", "a list of image family that target image want to test belonged to, split by ',', at least one provided")
+	imageProject   = flag.String("image_project", "", "a list of project that target image want to test located in, split by ',', at least one provided")
+	testBinaryPath = flag.String("test_binary_path", "", "a list of binary path that going to execute in VM, plit by ',', at least one provided")
+)
+
+func main() {
+	flag.Parse()
+	var imageTests []*e2etest.ImageTest
+	imageTests = parseImageTestArg(imageTests)
+	e2eTest := e2etest.E2ETest{
+		WorkProject: *workProject,
+		GcsPath:     *gcsPath,
+		ImageTests:  imageTests,
+	}
+
+	var errs []error
+	var err error
+
+	ctx := context.Background()
+	var ws []*daisy.Workflow
+	ws, err = e2eTest.CreateWorkflows(ctx)
+	if err != nil {
+		createWorkflowErr := fmt.Errorf("workflow creation error: %s", err)
+		fmt.Println(createWorkflowErr)
+		errs = append(errs, createWorkflowErr)
```

Comment:
there's probably no use appending this if you just call os.exit right after

can you move the golang stuff to a separate PR